macrel/main.py

```diff
@@ -124,9 +124,9 @@ def validate_args(args):
         error_exit(args, '--file-output is only possible for `get-smorfs` command')
 
     if args.logfile:
-        if path.exists(args.logfile):
+        if path.exists(args.logfile) and args.logfile != '/dev/null':
```

Comment:
Just remove this check. It's almost impossible to think of every special case. I think over-writing the log is fine. I think we also do not check in the case of `file-output` being used.